testing-guidance/workflow-tests/physics/physics-lkg-workflows.md

```diff
@@ -83,37 +91,125 @@ Any project that has the following Gems enabled:
+|-------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Creating an entity with PhysX Character Controller component    | <ol><li>Create an entity with PhysX Character Controller based on each supported shape, add a mesh component to it for visibility.<li>Collide other PhysX entities with PhysX Character Controller entity to verify its collision.</ul></li><li>Verify behavior of collisions with PhysX Character Controller after assigning it with various Physics Materials.</li></ol> | <ul><li>PhysX character controller as a kinematic controller is not affected by gravity and outside forces but can still collide with other PhysX entities.</li><li>PhysX character controller can be assigned with all available shapes and these shapes dictate how other entities collide with PhysX Character Controller.</li><li>Shapes assigned to PhysX Character Controller can be modified with options found in the component.</li></ul>
```

Comment:
Step 1: Link off to supported shapes.
Step 2: Add detail to how to do this either by linking off to another workflow for example or adding sub-steps to give a brief example.
Step 3: Reword to actionable steps that allows verification language to move to Things to Watch Out For.